CONTRIBUTE.md

```diff
@@ -0,0 +1,89 @@
+# Contribution Guidelines
+
+<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [Contribution Guidelines](#contribution-guidelines)
+	- [Issues](#issues)
+		- [Technical issues](#technical-issues)
+		- [Routing issues](#routing-issues)
+		- [Feature requests](#feature-requests)
+	- [Pull Requests](#pull-requests)
+		- [Pull request guidelines](#pull-request-guidelines)
+		- [Contributing code](#contributing-code)
+
+<!-- /TOC -->
+
+**Firstly, thanks for using the ORS and contributing to making it better for everyone.**
+
+OpenRouteService has been around for over ten years, and as such it has seen many changes and adoption of new methodologies and techniques, as well as the coming and going of multiple developers. To try and keep the OpenRouteService codebase clean and stable, there are a few guidelines that we try to follow.
+
+## Issues
+
+We accept technical issues, routing issues and feature requests. Common questions should be asked at [ask.openrouteservice.org](https://ask.openrouteservice.org).
+
+Please search for issues before creating [a new one](https://github.com/GIScience/openrouteservice/issues/new).
+
+### Technical issues
+
+If you encounter a bug, please make sure to be as descriptive as possible, i.e.:
+
+- operating system (in case of self-hosting)
+- request URL (incl. parameters if POST)
+- expected outcome
+- actual outcome (e.g. JSON output)
+- fenced code around the bug, if known
+
+### Routing issues
+
+If you encounter weird or unexpected behavior during routing, please make sure to rule out expected behavior based on road attributes from e.g. [openstreetmap.org](https://openstreetmap.org/query), such as unexpected one-way streets or access restrictions. Find an overview of our tagging filters for all transportation profiles in our [wiki](https://github.com/GIScience/openrouteservice/wiki/Tag-Filtering).
+
+Information to include:
+
+- request URL (incl. parameters if POST)
+- expected outcome
+- actual outcome (e.g. JSON output)
+- screenshots where applicable (e.g. from our [maps](https://maps.openrouteservice.org))
+
+### Feature requests
+
+Here you can be creative, but still descriptive. Make sure to describe the current behavior, your desired behavior and **to give actual use cases**. That makes it whole easier for us to prioritize.
+
+## Pull Requests
+
+We :heart: pull requests! We also aspire to make our commit history cleaner and more sustainable, benefiting our contributors and us maintainers.
+
+### Pull request guidelines
+
+We'd like all pull requests to adhere to the following rules:
+
+- a PR has to **close an issue**. If there is none yet for yours, please [create one](https://github.com/GIScience/openrouteservice/issues/new)
+- branch off [developemnt](https://github.com/GIScience/openrouteservice/tree/development)
```

Comment:
Spelling - development

Reply:
fixed

Reply:
but have you? :smirk: still says developemnt

Reply:
uff, right..